schemas/json/block.json

```diff
@@ -897,91 +897,98 @@
 			]
 		},
 		"variations": {
-			"type": "array",
 			"description": "Block Variations is the API that allows a block to have similar versions of it, but all these versions share some common functionality.",
-			"items": {
-				"type": "object",
-				"required": [ "name", "title" ],
-				"additionalProperties": false,
-				"properties": {
-					"name": {
-						"type": "string",
-						"description": "The unique and machine-readable name."
-					},
-					"title": {
-						"type": "string",
-						"description": "A human-readable variation title."
-					},
-					"description": {
-						"type": "string",
-						"description": "A detailed variation description."
-					},
-					"category": {
-						"description": "A category classification, used in search interfaces to arrange block types by category.",
-						"anyOf": [
-							{
-								"type": "string"
+			"oneOf": [
+				{
+					"type": "string"
```

Comment:
Do you think it is worth adding a `description` property here to better document the expected usage of this?

Reply:
Yeah, that's a good point. I thought it might collide with the `description` a couple lines above (i.e. which applies to both `oneOf` options), but it seems to be allowed by the JSON schema spec, so I'll add it.

Reply:
be8a9c2ba35